### DISCUSSION TITLE
Search vs. Show

### DISCUSSION THREAD

**Discussion by mthmulders** [mthmulders/mcs#32] [Ideas]
Currently, `mcs` will output a pom.xml snippet when it finds exactly one result. This could even happen with `-l 3` specified - when you ask for last three but there's just one result :-).

I am not sure if this is how I'd like to see it moving forward. Some ideas:
1. Maybe it would make more sense to have a `--maven` (and `--gradle` and the like) switch that would show the relevant snippets; additionally (or with another CLI flag, maybe `-c`/`--copy`) it could also copy it to the clipboard.
1. For consistency reasons, even when there's one result, it could still be printed in tabular form.
1. We may need a new command, "select", that helps you select an artifact interactively (not sure if PicoCLI can do that) and eventually copy it to the clipboard in the syntax the user has chosen.

I'd love to hear your ideas around this topic!

**Comment by rotty3000** [comment]
A recent thread on maven slack showed [console ui](https://youtu.be/6dB3CyOX9rU) and the second prompt was a interactive selection. I would be really cool IF when you get multiple search results to put up an interactive selection like that.
